_IBC/archway-osmosis.json

```diff
@@ -55,6 +55,21 @@
       "tags": {
         "status": "live"
       }
+    },
+    {
+      "chain_1": {
+        "channel_id": "channel-*",
+        "port_id": "wasm.*"
```

Comment:
you must open specific port/channels and publish them all here. Then you can use wildcard in your relayer config to process packets on all ports/channels

Reply:
seems like upstream chain registry does not have a standard way of publishing wild cards, so I will accept it like this for now and change in the future if there a different standard introduced upstream